actioncable/lib/action_cable/connection/stream.rb

```diff
@@ -29,8 +29,9 @@ def shutdown
       def write(data)
         return @rack_hijack_io.write(data) if @rack_hijack_io
         return @stream_send.call(data) if @stream_send
-      rescue EOFError
+      rescue EOFError, Errno::ECONNRESET
         @socket_object.client_gone
+        raise
```

Comment:
It won't actually get emitted, because of the ready_state... right? :confused:

But, are these "error"-worthy? We heard about it through an exception, but client disappearance is a pretty ordinary occurrence: if we've already handled it, maybe that's all that needs to happen.